src/Microsoft.OData.Core/Metadata/MetadataUtils.cs

```diff
@@ -174,22 +174,7 @@ internal static IList<IEdmOperation> CalculateBindableOperationsForType(IEdmType
             List<IEdmOperation> operationsFound = new List<IEdmOperation>();
             foreach (IEdmOperation operation in operations)

```

Comment:
First, check to see if the returned operations is an IList<IEdmOperation> (as it will be in most cases.  If so, return it directly, rather than creating a copy of the list to return.